src/web/app/user-join-page.component.spec.ts

```diff
@@ -142,27 +156,20 @@ describe('UserJoinPageComponent', () => {
   });
 
   it('should stop loading if user is not logged in', () => {
```

Comment:
We'll need to use a different test strategy here. It is true that the page stops loading, but more importantly, we need to test that there is a redirect to the login page.